src/main/java/msg/StdMsgs.java

```diff
@@ -0,0 +1,36 @@
+package msg;
+/** This enum class represents a collection of standard messages or responses Sir Duke will say.
+ * His logo is also stored here
+ */
+public enum StdMsgs {
+    WELCOME(new Msg("Greetings Sir/Mdm, I am Sir sirduke, at your service\n" +
```

Comment:
I like how you have split the long message text into two, but would it be more easy to follow if you split the line before the operator?